lib/rules/media-feature-parentheses-space-inside/index.js

```diff
@@ -31,50 +31,62 @@ const rule = function(expectation) {
       // will be at atRule.raws.params.raw
       const params = _.get(atRule, "raws.params.raw", atRule.params);
       const indexBoost = atRuleParamIndex(atRule);
+      const errors = [];
```

Comment:
Can we rename this to `violations`, please?

It's a convention we've adopted to:

- help distinguish between things caught by the linter and actual errors, for example thrown by the parsers.
- act as an intermediary before those things are categorised into a particular severity e.g. "error" or "warning".

Reply:
Agreed. No problem at all.

Reply:
Done!